src/vos/vos_aggregate.c

```diff
@@ -1359,10 +1363,40 @@ unmark_removals(struct agg_merge_window *mw, const struct agg_phy_ent *phy_ent)
 	}
 }
 
+static void
+dump_trace(struct vos_agg_param *agg_param)
+{
+	vos_iter_entry_t *entry;
+	int               i;
+	int               last;
+
+	if (agg_param->ap_trace_count == 0)
+		return;
+
+	D_ERROR("Assertion will trigger, dumping the evt_trace\n");
```

Comment:
It's better to print the 'ap_trace_count' as well, so that we could know if any trace was truncated when EV_TRACE_MAX traces are printed.

Reply:
Will add that.